platform/viewer/src/connectedComponents/ViewerRetrieveStudyData.js

```diff
@@ -218,6 +219,7 @@ function ViewerRetrieveStudyData({
     );
 
     setStudies([...studies, study]);
+    setIsStudyLoaded(true);
```

Comment:
I get it, but I think this can be cleaner. Probably not blocking, but worth thinking about.